src/archive.cpp

```diff
@@ -77,6 +77,24 @@ namespace zim
     }
   }
 
+
+
+  entry_index_type Archive::getMediaCount() const
+  {
+    try {
+      return countMimeType(
+        getMetadata("Counter"),
+        [](const std::string& mimetype) {
+          return mimetype.find("image/") == 0 ||
+                 mimetype.find("video/") == 0 ||
+                 mimetype.find("audio/") == 0;
+        }
+      );
+    } catch(const EntryNotFound& e) {
+      return 0;
+    }
```

Comment:
I wonder if handling the absence of the `Counter` metadata item this way is the right thing to do.

Reply:
Yes, it was what was made on libkiwix but we don't have to follow the same convention.
Now we throw the `EntryNotFound`.